currency/addbalance.cc.lua

```diff
@@ -0,0 +1,39 @@
+{{/*
+		Simple Currency System By GasInfinity
+        
+        Recommended Trigger: Regex: "^-(addbalance|addbal)"
+        
+        Adds Money To a User,Also Removes It, Usage "-addbalance <User/Member> <Amount>" 
+*/}}
+
+{{/*CONFIGURATION START*/}}
+
+{{ $currency := "💰" }} {{/*Currency Emoji/Name*/}}
+
+{{/*CONFIGURATION END*/}}
+
+
+
+
+{{/*MAIN CODE !!!*/}}
+
+
+{{$args := parseArgs 2 "Syntax is -addbalance <User> <Amount>"
+    (carg "string" "User To Send Balance")
+    (carg "int" "Balance to Add")}}
+
+{{$user := (userArg ($args.Get 0))}}

```

Comment:
This is redundant and is also prone to error, for example, `-addbalance "Hello world!" 123` is not considered invalid, but it will cause errors later on as `$user` does not exist -> `$user.ID` will error.

Instead, consider the following:
Note that we are using `member` carg instead of `userid` or `user`. This is because both have their own faults:
- `user` does not match on user IDs.
- `userid` does not validate the ID, i.e `123` is valid input.

`member` solves both of these issues and provides a member struct for you to use. You can then use `$member.User` to obtain the underlying user.
```suggestion
{{$args := parseArgs 2 "Syntax is -addbalance <User> <Amount>"
    (carg "member" "User To Send Balance")
    (carg "int" "Balance to Add")}}

{{$user := ($args.Get 0).User}}
```

Reply:
Done